.env.example

```
@@ -0,0 +1,14 @@
+﻿NODE_ENV=development
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)